src/con_utils.ts

```diff
@@ -197,47 +198,35 @@ function parseConnectDsnAndArgs({
       }
     }
 
-    if (parsed.auth) {
-      const [puser, ppass] = parsed.auth.split(":");
-      if (puser && user == null) {
-        user = puser;
-      }
-      if (ppass && password == null) {
-        password = ppass;
-      }
+    if (parsed.username && user == null) {
+      user = parsed.username;
+    }
+    if (parsed.password && password == null) {
+      password = parsed.password;
```

Comment:
Hm, this seems to be unrelated to this PR. I fear we might be accidentally reverting new-ish changes.

Reply:
Ah, I see, nvm